measures/op02n12a.measures.resource.json

```diff
@@ -22,6 +128,6 @@
       "name": "Johannes Hentschel",
       "@id": "https://orcid.org/0000-0002-1986-9545"
     },
-    "softwareVersion": "2.5.4"
+    "softwareVersion": "0.0.post1.dev1+g6d954be"
```

Comment:
Uh oh, not sure why this version of ```ms3``` is here....maybe this is why weird stuff happened? I can't believe it, since I fresh installed ```ms3``` again specifically for this task. @johentsch